docs/_documentations/intellij-codechange.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: docs
+title: Making a code change with IntelliJ
+description: Making a code change with IntelliJ
+keywords: changing, build, building, builds, automatic, redeploy, redeploying, redeploys, application, applications, save, saving, saves, file, feature, edit, editing, edits
+duration: 1 minute
+permalink: intellij-codechange
+type: document
+---
+
+# Making a code change with IntelliJ
+Codewind automatically builds and redeploys your application whenever you make a code change and save the file. To see this feature in action, make a change to a file:
+1. From a project, open the file that you want to edit, for example, **Project1**>**src**>**main**>**java**>**application**>**rest**>**v1**>**example**.
+2. Edit your file.
+3. Save your changes. By default, IntelliJ automatically saves your changes. However, if you manually save your changes, you can help Codewind detect changes more quickly.
+4. Open the application again. Your changes appear.
+
+Next step: [Deploying Codewind remotely](remote-overview.html)
```

Comment:
@jopit Is linking to the remove overview a good link for IntelliJ, or should we avoid a link here for now? Thanks!

Reply:
We should avoid linking to the remote overview for now -- IntelliJ support for remote has not been implemented yet.